src/data/finset/pointwise.lean

```diff
@@ -408,18 +290,55 @@ coe_injective.semigroup _ coe_mul
 protected def comm_semigroup [comm_semigroup α] : comm_semigroup (finset α) :=
 coe_injective.comm_semigroup _ coe_mul
 
+section mul_one_class
+variables [mul_one_class α]
+
+/-- `finset α` is a `mul_one_class` under pointwise operations if `α` is. -/
+@[to_additive "`finset α` is an `add_zero_class` under pointwise operations if `α` is."]
+protected def mul_one_class : mul_one_class (finset α) :=
+coe_injective.mul_one_class _ (coe_singleton 1) coe_mul
+
+localized "attribute [instance] finset.mul_one_class finset.add_zero_class finset.semigroup
+  finset.add_semigroup" in pointwise
```

Comment:
Is `finset.comm_semigroup` missing here?

Reply:
Oh turns out it was missing originally as well. Really looking forward to automated localized instance declaration...